stable/aurora-platform/charts/aurora-core/templates/trust-manager/bundle.yaml

```diff
@@ -0,0 +1,53 @@
+{{ if .Values.components.trustManager.enabled -}}
+
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: "{{ .Values.global.cluster }}-trust-manager-bundle"
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: {{ .Values.global.project }}
+  source:
+    chart: {{ .Values.global.raw.helm.chart }}
+    repoURL: {{ default .Values.global.helm.repository .Values.global.raw.helm.repository }}
+    targetRevision: {{ .Values.global.raw.helm.targetRevision }}
+    plugin:
+      env:
+        - name: HELM_RELEASE_NAME

```

Comment:
We do we use `env.name: HELM_RELEASE_NAME` some places and not others?

Reply:
I think this is related to when we use the plugin for secret manager (ArgoCD Vault Plugin) we can't use the normal method and instead have to use this naming.

https://github.com/argoproj-labs/argocd-vault-plugin/issues/336#issuecomment-1103945205

Reply:
Oh good point we arent i fixed tihs!